src/main/java/nl/tudelft/pixelperfect/event/PlasmaLeak.java

```diff
@@ -0,0 +1,39 @@
+package nl.tudelft.pixelperfect.event;
+
+import com.jme3.scene.Geometry;
+
+/**
+ * Created by woute on 5/10/2016.
+ */
+public class PlasmaLeak extends Event {
```

Comment:
I suggest we make the names of Event subtypes contain the suffix 'Event', e.g. PlasmaLeakEvent.